example/client.rb

```diff
@@ -213,46 +213,52 @@ def create_and_fund_transaction_example
   end
 
   def webhook_parse_example
-    webhook_content = <<-JSON
-      {
-        "webhook": "85d11cf4-d4d6-46e4-ab7d-91355e80392a",
-        "event": "transaction.created",
-        "object": {
-          "input_amount": 21.59,
-          "input_currency": "GBP",
-          "id": "cd390e83-a9de-404d-a660-4da0f1342c04",
-          "metadata": {},
+    headers = {
+      "Authorization-Nonce": "<authorization-nonce>",
```

Comment:
good point made in the other PR: this needs to be a symbol if we're using symbols